### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by Ellon-M**
# Changes in this pull request
- Added media queries for work and about sections to resize the grids in different screen sizes.
- Added additional media queries for the contact form for larger screens.
- Used cover images for desktop screens as per the Figma design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -120,13 +125,39 @@ <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
...
+          </a>
+        </article>
+        <article class="work">
+          <img src="./media/printing.png" alt="first project screenshot" class="work-image">
+          <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+          <ul class="language-tags">
+            <li class="tag">Ruby on rails</li>
+            <li class="tag">JavaScript</li>
+            <li class="tag">CSS</li>
+            <li class="tag">HTML</li>
```
Comment (Yerimah) [inline index.html]: - [ ] Hey @Ellon-M, 👋 you have done well in implementing most of the project requirements. I would kindly suggest you try to make your design of the `tech-stack class` align as close as possible with the design of your chosen Figma template.

> Figma Template

your shoot

> Your design
newshoot

File: index.html [hunk 2]
```diff
@@ -10,56 +10,61 @@
...
+          </li>
+          <li class="icon">
+            <a href="https://angel.co/u/ellon-mordecai">
+              <img src="https://img.icons8.com/ios/32/172B4D/angelist.png"/>
+            </a>
+          </li>
+        </ul>
+        <div id="main-button">
+          <img src="https://img.icons8.com/ios/45/172B4D/circled-chevron-down.png"/>
+        </div>
```
Comment (Yerimah) [inline index.html]: I would suggest you add the `alt` attribute to the `` elements of your social icons. The required alt attribute specifies an alternate text for an image if the image cannot be displayed. This improves your portfolio's accessibility

File: index.html [hunk 3]
```diff
@@ -222,7 +261,6 @@ <h4>mordecellon@gmail.com</h4>
       <div class="contact-bar">
         <img class="end-line" src="media/Shape.png" alt=" end line icon" />
       </div>
```
Comment (Yerimah) [inline index.html]: Please make sure to eliminate white spaces on your website. There is a white space underneath your footer icons.  

> Your design

spaces

File: index.html [hunk 4]
```diff
@@ -176,7 +207,15 @@ <h3 id="form-heading">
         </h3>
         <form action="https://formspree.io/f/xyyvbjao" method="post">
```
Comment (Yerimah) [inline index.html:208]: Please make sure to stick with the design specifications provided on the Figma template (e.g., font, colors, images, text, margins). Kindly adjust the font size and width of the paragraph beside your contact form to align with your chosen Figma design template.

> Your design
newcap contact

> Figma design template
newcap

**Review by Yerimah — CHANGES_REQUESTED** [review]
# Status: Required Changes 🔴 

Hi @Ellon-M, 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4d634a5755566852566e493378656f516f6d2f67697068792e676966

## Highlights
- No linter errors ✔️ 
- PR title and summary descriptive ✔️ 
-  GitHub flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (fherrerao) [inline index.html]: - [ ] Good job so far 🌜, as [requested](https://github.com/Ellon-M/portfolio/pull/7#discussion_r1014536405) by the previous reviewer, kindly, your design should sticks as much as possible with Figma design, please, fix it.

Your design:

image

Figma design:

image

File: index.html [hunk 5]
```diff
@@ -10,56 +10,61 @@
...
-      <div id="main-button">
-        <img src="https://img.icons8.com/ios/45/172B4D/circled-chevron-down.png"/>
-      </div>
-    </section>
+          </p>
+        </div>
+        <ul id="social-icons">
+          <li class="icon">
+            <a href="https://github.com/Ellon-M">
+              <img src="https://img.icons8.com/windows/32/172B4D/github.png"/>
```
Comment (fherrerao) [inline index.html]: - [ ] Please, remember to perform the changes [requested](https://github.com/Ellon-M/portfolio/pull/7#discussion_r1014538520) by the previous reviewer 🎲, add `alt` attribute for every `` tag, it represents an alternate text when an image can't be displayed 🎃.

**Review by fherrerao — CHANGES_REQUESTED** [review]
Hi @Ellon-M, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

So Close Snl GIF by Saturday Night Live
 ## Highlights

✔️  No linter errors.
✔️  You used the correct GitHub-Flow.
✔️  Good README
✔️  Good Commit Messages

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ellon-M** [comment]
Hello, 

All the requested changes have been made. If anything is missing, please let me know.

**Review by Debas-31 — APPROVED** [review]
> ## Hi @Ellon-M  :wave:, 

# Status: Approved :heavy_check_mark:  :heavy_check_mark: :heavy_check_mark: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Commit message :heavy_check_mark:
- PR title and summary :heavy_check_mark: 
- Responsiveness :heavy_check_mark: 
- Figma template usage :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**